File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/sqlserver/BoolType.kt
package dev.typr.foundations.docs.sqlserver

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class BoolType {
    //start
    val bitType: SqlServerType<Boolean> = SqlServerTypes.bit
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/BoolType.scala
package dev.typr.foundations.docs.sqlserver

import dev.typr.foundations.{SqlServerType, SqlServerTypes}

@SuppressWarnings(Array("unused"))
object BoolType:
  //start
  val bitType: SqlServerType[java.lang.Boolean] = SqlServerTypes.bit
  //stop
